projects/plugins/jetpack/modules/publicize/publicize.php

```diff
@@ -1,28 +1,51 @@
-<?php
+<?php //phpcs:ignore WordPress.Files.FileName.InvalidClassFileName
+/**
+ * Publicize_Base class.
+ *
+ * @package automattic/jetpack-publicize
+ *
+ * @since-jetpack $$next-version$$ Moved to the jetpack-publicize package.
```

Comment:
We're not actually doing this yet. Does it still need this @since-jetpack line?

Reply:
Nope, that was a mistake from copying the previous PR. Fixed in #9c97606.